python/py-wmctrl/Portfile

```diff
@@ -0,0 +1,26 @@
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-wmctrl
+version             0.3
+platforms           darwin
+license             BSD
+maintainers         nomaintainer
+
+description         A tool to programmatically control windows inside X
+long_description    ${description}
+
+homepage            https://bitbucket.org/antocuni/wmctrl/src/default/
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
```

Comment:
the ```master_sites``` and ```distname``` are set automatically by the python PG; so if the package is hosted on PyPI you do not need these lines in the Portfile

Reply:
Done